lib/commons/aria/index.js

```diff
@@ -388,7 +387,7 @@ lookupTables.role = {
 		owned: null,
 		nameFrom: ['author', 'contents'],
 		context: ['row'],
-		implicit: ['td', 'th']
+		implicit: ['th']
```

Comment:
What's the reason for removing `td` here?

Reply:
https://www.w3.org/TR/html-aria/#TD - An td never has an implicit role of gridcell following the current spec.